runtime/doc/usr_05.txt

```diff
@@ -47,7 +47,7 @@ The vimrc file can contain all the commands that you type after a colon.  The
 simplest ones are for setting options.  For example, if you want Vim to always
 start with the 'incsearch' option on, add this line your vimrc file: >
```

Comment:
Change this line instead. It used to be `'ignorecase'` before <https://github.com/neovim/neovim/commit/09f3e62bc623dd3cbcbc708b019b55a8b9449de7>

Reply:
Ah, wondered which way would be better, should've just `blame`d.. Now, would it be preferred to do this PR all new over a second commit reverting the first and changing the correct place?

Sorry, didn't want to cost you this much trouble for such a simple typo fix.